app/_src/policies/meshloadbalancingstrategy.md

```diff
@@ -61,6 +61,15 @@ Advanced locality-aware load balancing provides a powerful means of defining how
       - **`zones`** - names of zones
   - **`failoverThreshold.percentage`** - (optional) defines the percentage of live destination dataplane proxies below which load balancing to the next priority starts. Default: 50%
 
+#### Zone Egress support
+
+Using Zone Egress Proxy in multizone deployment poses certain limitations for this policy. When configuring `MeshLoadbalancingStrategy` with Zone Egress you can only use `Mesh` as a top level targetRef. This is because we don't differentiate requests that come to Zone Egress from different clients, yet. 
+
+Moreover, Zone Egress is a simple proxy that uses a long living L4 connection between Zone Ingress. Because of this, when new `MeshLoadbalancingStrategy` with locality awareness is configured, we won’t refresh connections and locality awareness will apply only to new connections. 
```

Comment:
> between Zone Ingress.

This does part is missing something, I think you need a plural after "between" or `and`.

```suggestion
Moreover, Zone Egress is a simple proxy that uses a long-living L4 connection to Zone Ingress. Consequently, when a new `MeshLoadbalancingStrategy` with locality awareness is configured, connections won’t be refreshed, and locality awareness will apply only to new connections.
```